### PR TITLE
feat-Added badge component

### PR DESCRIPTION
**Pull request by krishanu2911**
Added  variations of the badge component on icon and avatar (online, offline, dnd)
Screenshot (412)

### CODE REVIEW

File: CSS/components/badge.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+}
+.badge-value{
+    display: flex;
+    justify-content: center;
+    padding: 0.5rem;
+    align-items: center;
+    position: absolute;
+    background-color: var(--primary-color);
+    color: var(--background-color);
+    width: 1rem;
```
Comment (vivekmehto) [inline CSS/components/badge.css]: You can increase badge width for icons. That 10 is not clearly visible. make badge width for at least 2 digit numbers fit in.

**Review by vivekmehto — COMMENTED** [review]
looks good just one suggestion for the icon with a badge.

**Review by rahulcs754 — COMMENTED** [review]
all look good 
Notes: i think you should  set min-width for number counter

File: CSS/components/badge.css (hunk 1 above)
Comment (krishanu2911) [reply CSS/components/badge.css]: Thanks for pointing out adjusted the size

Comment (krishanu2911) [reply CSS/components/badge.css]: Screenshot (414)

**Review by vivekmehto — APPROVED** [review]
Looks perfect now.

**Pull request merged**